README.rst

```diff
@@ -254,101 +254,18 @@ Other non-official Python wrappers for GMT (not maintained):
-      - >=1.22
-    * - `v0.9.0 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.9.0>`_
-      - `v0.9.0 Documentation <https://www.pygmt.org/v0.9.0>`_
-      - >=6.3.0
-      - >=3.8
-      - >=1.21
-    * - `v0.8.0 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.8.0>`_
-      - `v0.8.0 Documentation <https://www.pygmt.org/v0.8.0>`_
-      - >=6.3.0
-      - >=3.8
-      - >=1.20
-    * - `v0.7.0 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.7.0>`_
-      - `v0.7.0 Documentation <https://www.pygmt.org/v0.7.0>`_
-      - >=6.3.0
-      - >=3.8
-      - >=1.20
-    * - `v0.6.1 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.6.1>`_
-      - `v0.6.1 Documentation <https://www.pygmt.org/v0.6.1>`_
-      - >=6.3.0
-      - >=3.8
-      - >=1.19
-    * - `v0.6.0 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.6.0>`_
-      - `v0.6.0 Documentation <https://www.pygmt.org/v0.6.0>`_
-      - >=6.3.0
-      - >=3.8
-      - >=1.19
-    * - `v0.5.0 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.5.0>`_
-      - `v0.5.0 Documentation <https://www.pygmt.org/v0.5.0>`_
-      - >=6.2.0
-      - >=3.7
-      - >=1.18
-    * - `v0.4.1 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.4.1>`_
-      - `v0.4.1 Documentation <https://www.pygmt.org/v0.4.1>`_
-      - >=6.2.0
-      - >=3.7
-      - >=1.17
-    * - `v0.4.0 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.4.0>`_
-      - `v0.4.0 Documentation <https://www.pygmt.org/v0.4.0>`_
-      - >=6.2.0
-      - >=3.7
-      - >=1.17
-    * - `v0.3.1 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.3.1>`_
-      - `v0.3.1 Documentation <https://www.pygmt.org/v0.3.1>`_
-      - >=6.1.1
-      - >=3.7
-      -
-    * - `v0.3.0 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.3.0>`_
-      - `v0.3.0 Documentation <https://www.pygmt.org/v0.3.0>`_
-      - >=6.1.1
-      - >=3.7
-      -
-    * - `v0.2.1 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.2.1>`_
-      - `v0.2.1 Documentation <https://www.pygmt.org/v0.2.1>`_
-      - >=6.1.1
-      - >=3.6
-      -
-    * - `v0.2.0 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.2.0>`_
-      - `v0.2.0 Documentation <https://www.pygmt.org/v0.2.0>`_
-      - >=6.1.1
-      - 3.6 - 3.8
-      -
-    * - `v0.1.2 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.1.2>`_
-      - `v0.1.2 Documentation <https://www.pygmt.org/v0.1.2>`_
-      - >=6.0.0
-      - 3.6 - 3.8
-      -
-    * - `v0.1.1 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.1.1>`_
-      - `v0.1.1 Documentation <https://www.pygmt.org/v0.1.1>`_
-      - >=6.0.0
-      - 3.6 - 3.8
-      -
-    * - `v0.1.0 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.1.0>`_
-      - `v0.1.0 Documentation <https://www.pygmt.org/v0.1.0>`_
-      - >=6.0.0
-      - 3.6 - 3.8
-      -
+Minimum Supported GMT/Python/NumPy Versions
```

Comment:
This section is not shown on the index page (https://pygmt-dev--2862.org.readthedocs.build/en/2862/index.html) but is shown in the README file.